lib/fluent/plugin/filter_process_redfishalert.rb

```diff
@@ -70,7 +71,11 @@ def callRedfishGetAPI(host, resourceURI)
     end
 
     def getMachineIdentifier(host)
-      res = callRedfishGetAPI(host, @hwtDeviceURI[hardware])
+      begin
+        res = callRedfishGetAPI(host, @hwtDeviceURI[hardware])
+      rescue NoMethodError => e
```

Comment:
Can you please add a comment specifying what is expected for SDFLEX and SDFLEX 280, will be helpful in future.

Reply:
Like which endpoint is meant for which kind of system? Added.

Reply:
Thanks